### PR TITLE
#158863118 Fix Navbar responsiveness

### PR DESCRIPTION
**Pull request by missating**
**What does this PR do?**

It fixes the responsiveness of the navigation bar and the asset pane on large screen

**Description of Task to be completed?**
   - fix navbar responsiveness
   - centralize the content of the settings page
    - fix overflow on the x-axis of the webpage
    - add min-height to the dashboard container

**How should this be manually tested?**
 - `yarn start:dev` to open the web browser on `port 3000` 
- On small devices change your display options to `scaled` with more space to make sure it's responsive or view the webpage on a desktop

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#158863118](https://www.pivotaltracker.com/story/show/58863118)

**Screenshots (if appropriate)**
Default display for laptops:
screen shot 2018-07-11 at 3 12 27 pm

Scaled display with more space or desktop:
screen shot 2018-07-11 at 3 15 27 pm

**Questions:**

N/A

### CODE REVIEW

**Comment by andela-kakpobome** [comment]
activodev andela com3000dashboard

**Comment by missating** [comment]
@bourgeois247 I get the part about the margin would fix that.  

But the sidebar in question is it the asset pane div? and at what point is it visible? when you try to scroll to the right or just looking at the webpage?

**Comment by andela-kakpobome** [comment]
1. Yes, asset pane div
2. Without scrolling, try on a monitor and see

**Comment by missating** [comment]
Okay. Seen, working on it now.

**Comment by missating** [comment]
@bourgeois247 PR has been updated.

File: src/components/AddAssetPane/AddAssetPane.scss [hunk 1]
```diff
@@ -75,3 +75,10 @@
 .asset-page-select-width {
   width: 22.075rem;
 }
+
+
+// @media only screen  and (min-width : 1441px) {
```
Comment (andela-kakpobome) [inline src/components/AddAssetPane/AddAssetPane.scss]: Why do you have commented code here?

Comment (missating) [reply src/components/AddAssetPane/AddAssetPane.scss]: Fixed.

**Pull request merged**